ManatiCR/vitaminau
# VIT-37: Form beneficiaries.

# Task [VIT-37](https://manati.atlassian.net/browse/VIT-37?atlOrigin=eyJpIjoiNDhhZDVlZGE1ZTM5NDhlOWE4Y2Y2ZTY5OTQ0ZmM2NmIiLCJwIjoiaiJ9)
Creation of the beneficiary section of the forms.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following URL of the forms:
  - http://vitaminau.docker/form/webform-entrepreneurship-support.
  - http://vitaminau.docker/form/webform-licensed-technologies
  - http://vitaminau.docker/form/work-in-progress
  - http://vitaminau.docker/form/webform-installations-use
  - http://vitaminau.docker/form/sale-of-products
- Once on these pages, you must go to the section of beneficiaries and you must see the following fields:
  - [VIT-37](https://docs.google.com/spreadsheets/d/1g4bC1rd_48hHrhUQYUrL_A7Xa9Pc4C5U7gqaI5eaw6M/edit#gid=1641375975)

# Acceptance criteria
- Only authenticated users can access the form.
- The field called "Sector De Actividad Económica"  must load values ​​from the webform_options called options_economy_activity.
- The field called "Sector of economic activity" must be separated from the area of ​​the field called "Type of beneficiary".
- Character count in the following fields:
  - Sector de actividad económica -> No estoy seguro -> Comentario sobre sector de actividad económica
  - Tipo de beneficiario -> Otro -> Detalle el tipo de beneficiario
  - Persona individual -> Persona especifica -> Persona individual específica
  - Comunidad -> Detalle comunidad
- The following fields are not required
  - Detalle comunidad
